### ISSUE TITLE
flutter_embedded example not working

### ISSUE THREAD

**Issue by MarkOSullivan94** [schultek/jaspr#339]
**Description**

I wanted to check out how flutter_embedding works with Jaspr and after getting the example project to run it doesn't embed Flutter like in the demo video

<img width="1728" alt="Screenshot 2024-11-22 at 20 30 45" src="https://github.com/user-attachments/assets/7430d6e3-22c0-4949-922b-4a01f31a4123">

**Additional Context**

jaspr: v0.16.1

**pubspec.yaml**

This is the pubspec.yaml file in the example project 

```yaml
name: flutter_embedding_demo
description: A demo of embedding a flutter app into a jaspr website.
version: 0.1.3
publish_to: none

environment:
  sdk: '>=3.0.0-0 <4.0.0'

dependencies:
  firebase_core: ^3.8.0
  flutter:
    sdk: flutter
  flutter_riverpod: ^2.6.1
  jaspr: ^0.16.1
  jaspr_flutter_embed: ^0.4.1
  jaspr_riverpod: ^0.3.15
  riverpod: ^2.6.1

dev_dependencies:
  build_runner: ^2.4.13
  flutter_test:
    sdk: flutter
  flutter_lints: ^5.0.0
  jaspr_builder: ^0.16.1
  jaspr_web_compilers: ^4.0.10+1

flutter:
  uses-material-design: true
  assets:
    - assets/dash.png

jaspr:
  mode: server
```

**Comment by schultek** [comment]
There was a layout bug in jaspr_flutter_embed. Fixed in the newest version `0.4.2`.

**Comment by MarkOSullivan94** [comment]
Thanks for the swift fix @schultek!
